src/applications/vaos/components/calendar/CalendarOptions.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import classNames from 'classnames';
+import CalendarRadioOption from './CalendarRadioOption';
+import CalendarCheckboxOption from './CalendarCheckboxOption';
+import { isDateOptionPairInSelectedArray } from './../../utils/calendar';
+
+export default function CalendarOptions({
+  currentlySelectedDate,
+  additionalOptions,
+  handleSelectOption,
+  optionsError,
+  selectedDates,
+  selectedCellIndex,
+}) {
+  const selectedDateOptions = additionalOptions?.getOptionsByDate(
+    currentlySelectedDate,
+  );
+
+  if (selectedDateOptions) {
+    const fieldName = additionalOptions.fieldName;
+
+    const maxCellsPerRow = 4;
+    const middleCellIndex = 2;
+    const beginningCellIndex = [0, 1];
+    const endCellIndexes = [3, 4];
+
+    // If list of items is won't fill row, align items closer to selected cell
+    const cssClasses = classNames(
+      'vaos-calendar__options',
+      selectedDateOptions.length < maxCellsPerRow
+        ? {
+            'usa-input-error': optionsError,
+            'vads-u-justify-content--flex-start': beginningCellIndex.includes(
+              selectedCellIndex,
+            ),
+            'vads-u-justify-content--center':
+              selectedCellIndex === middleCellIndex,
+            'vads-u-justify-content--flex-end': endCellIndexes.includes(
+              selectedCellIndex,
+            ),
+          }
+        : null,
+    );
+
+    return (
+      <fieldset className="vads-u-order--last">
```

Comment:
@jbalboni I tried this out locally on Safari, and it's still skipping the `<fieldset>` when I'm moving by virtual cursor.

If, however, we add a `<div role="cell">` and move the CSS to that div and make it full width, Safari + VO honors the new "cell" and will cursor to it as expected.

Screen shot attached.

---
![Screen Shot 2019-12-17 at 3 46 12 PM](https://user-images.githubusercontent.com/934879/71037222-e5e76180-20e4-11ea-8822-bde196301ef3.png)

Reply:
Easy enough! Added